docs/generated/metrics/metrics.html

```diff
@@ -0,0 +1,1526 @@
+<table>
+<thead><tr><th>Layer</th><th>Metric</th><th>Description</th><th>Y-Axis Label</th><th>Type</th><th>Unit</th><th>Aggregation</th><th>Derivative</th></tr></thead>
```

Comment:
i don't think this should be in the backport, right?

Reply:
oops, removed. Thanks!